exporters/otlp/include/opentelemetry/exporters/otlp/otlp_exporter.h

```diff
@@ -8,18 +8,31 @@ namespace exporter
 {
 namespace otlp
 {
+/**
+ * Struct to hold OTLP exporter options.
+ */
+struct OtlpExporterOptions
+{
+  // The endpoint to export to. By default the OpenTelemetry Collector's default endpoint.
+  std::string endpoint = "localhost:55680";
+};
+
 /**
  * The OTLP exporter exports span data in OpenTelemetry Protocol (OTLP) format.
  */
 class OtlpExporter final : public opentelemetry::sdk::trace::SpanExporter
 {
 public:
   /**
-   * Create an OtlpExporter. This constructor initializes a service stub to be
-   * used for exporting.
+   * Create an OtlpExporter using all default options.
    */
   OtlpExporter();
 
+  /**
+   * Create an OtlpExporter using the given options.
+   */
+  OtlpExporter(OtlpExporterOptions options);
```

Comment:
I think this probably should be a const pointer?

Reply:
+1 for const ref